lib/ansible/modules/cloud/azure/azure_rm_storageaccountkeys_facts.py

```diff
@@ -0,0 +1,112 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2016 Amanvir Mundra, <@amanvirmundra>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {
+    'metadata_version': '1.1',
+    'status': ['preview'],
+    'supported_by': 'community'
+}
+
+DOCUMENTATION = '''
+---
+module: azure_rm_storageaccountkeys_facts
+
+version_added: "2.6"
+
+short_description: Retrieve Azure Storage Account Keys.
+
+description:
+    - Retrive keys for a Storage Account.
+
+options:
+    resource_group:
+        description:
+            - This is the name of the resource group for which you want to retrieve Keys
+        required: true
+        default: null
+    name:
+        description:
+            - Name of the Storage Account instance.
+        required: true
+        default: null
+        aliases:
```

Comment:
New modules shouldn't have aliases, plus the alias seems to be wrong as this is the name of the storage account instance.